index.html

```diff
@@ -83,95 +113,181 @@
+
+  // EVENT HANDLERS
+  d3.select('#year-range').on('change', function () {
+    options.yearRange = d3.event.target.value;
+    chart.update();
+    });
   });
 
-  function setup() {
-    // SVG
+  function Chart() { 
+    var chart = this; 
 
-    svg = d3.select('#chart')
+    // SVG
+    chart.svg = d3.select('#chart')
       .append('svg')
       .attr('width', width + margin.right + margin.left)
       .attr('height', height + margin.top + margin.bottom)
       .append('g')
       .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
-
+    
     // SCALES
-
-    x = d3.scale.linear()
-      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+    chart.x = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.costs; }))
       .range([0, width])
       .nice();
 
-    y = d3.scale.linear()
-      .domain(d3.extent(data, function (d) { return d.inflation; }))
+    chart.y = d3.scale.linear()
+      .domain(d3.extent(data, function (a) {return a.visits; }))
       .range([height, 0])
       .nice();
-
+    
     // AXES
-
-    // EVENT HANDLERS
-
-    d3.select('#year-range').on('change', function () {
-      options.yearRange = d3.event.target.value;
-      update();
-    });
+    var formatNumber = d3.format(".0f");
+
+    var xAxis = d3.svg.axis()
+      .scale(chart.x)
+      .tickFormat(formatCurrency)
+      .tickSize(height)
+      .orient("bottom"); 
+    
+    svg.append("g")
+      .append("text")
+      .attr("x", 190)
+      .attr("y", 580)
+      .style("text-anchor", "start")
+      .style("font-family", "Open Sans Condensed, Helvetica, sans-serif")
+      .style("font-weight", "bold")
+      .text("Standardized, Risk-Adjusted Per Capita Medicare Spending");
+    
+    var gx = svg.append("g")
+      .attr("class", "x axis")
+      .call(xAxis);
+    
+    gx.selectAll("g").filter(function(a) {return a;})
+      .classed("minor", true);
+    
+    var yAxis = d3.svg.axis()
+      .scale(chart.y)
+      .tickSize(width)
+      .orient("right");
+    
+    svg.append("g")
+      .append("text")
+      .attr("transform","rotate(-90)")
+      .attr("x", -250)
+      .attr("y", -60)
+      .attr("dy", ".71em")
+      .style("text-anchor", "middle")
+      .style("font-family", "Open Sans Condensed, Helvetica, sans-serif")
+      .style("font-weight", "bold")
+      .text("ER Visits per 100K");
+    
+    var gy = svg.append("g")
+      .attr("class", "y axis")
+      .call(yAxis);
+    
+    gy.selectAll("g").filter(function(a) {return a;})
+      .classed("minor", true);
+    
+    gy.selectAll("text")
+      .attr("x", -30);
+
+    chart.tooltip = d3.tip()
+      .attr("class", "tooltip")
+      .offset([-6,0])
+      .html(function (d) { return d.costs + ' in pc costs' ; }); 
+
+    chart.svg.call(chart.tooltip); 
+  
+    function formatCurrency(d) {
+      var s = formatNumber(d / 1e3);
+      return d === x.domain()[1]
+      ? "$" + s + " thousand"
+      : s;
+
```

Comment:
You're missing a curly brace here as well - need to close the formatCurrency function first. Then the next curly brace - below - is for closing the constructor function.